### DISCUSSION TITLE
bg not showing up in the bg list

### DISCUSSION THREAD

**Discussion by tofu111456** [KadeArchive/Kade-Engine#1383] [Q&A]
![image](https://user-images.githubusercontent.com/69342634/126058549-9d92957b-7580-4d2c-bc97-dfa6f4441118.png)
i tried to add my bg, so i coded it like the top. but when i compiled it, the bg didnt show up on the list

**Comment by TowsterBusiness** [comment]
You might need to hard code that in

**Comment by tofu111456** [comment]
> 
> 
> You might need to hard code that in

how do i do it then?

**Comment by flashintv** [comment]
dude add it in the '**stageList.txt**' stoopid

**Comment by tofu111456** [comment]
> 
> 
> dude add it in the '**stageList.txt**' stoopid

oh sry i didnt even think about it ;3;

**Comment by flashintv** [comment]
> > dude add it in the '**stageList.txt**' stoopid
> 
> oh sry i didnt even think about it ;3;

:trollface:

**Comment by tofu111456** [comment]
it works, but the character positions are kinda off. so i tried coding in the position things
but now this happened
![image](https://user-images.githubusercontent.com/69342634/126068316-3c8ee18a-09e0-4d9f-bb29-158e204a239c.png)
do i just need to use like the boyfriend.x thing

**Comment by tofu111456** [comment]
nvm, i tried using boyfriend. x, boyfriend.y, gf.x and gf,y and it worked lol

**Comment by tofu111456** [comment]
ok, now i tried to reposition them, but they just wont move to the correct spot. they will just end up at the bottom
how do i fix this

**Comment by HePpYPro** [comment]
keep trying changing the boyfriend.setposition but just try the y which is the level of height in which the character is.

Sussy Baka

**Comment by tofu111456** [comment]
now i tried to change the bg to my bg
but it just showed up as the stage

**Comment by ACardboardBox1** [comment]
> 
> 
> now i tried to change the bg to my bg
> but it just showed up as the stage

stages were unhardcoded to put them into charts.

**Comment by tofu111456** [comment]
> 
> 
> > now i tried to change the bg to my bg
> > but it just showed up as the stage
> 
> stages were unhardcoded to put them into charts.

what should i do then
